Content.Server/StationEvents/Events/NoosphericZapRule.cs

```diff
@@ -26,29 +26,25 @@ protected override void Started(EntityUid uid, NoosphericZapRuleComponent compon
     {
         base.Started(uid, component, gameRule, args);
 
-        var query = EntityQueryEnumerator<PotentialPsionicComponent, MobStateComponent>();
+        var query = EntityQueryEnumerator<PsionicComponent, MobStateComponent>();
 
-        while (query.MoveNext(out var psion, out var potentialPsionicComponent, out _))
+        while (query.MoveNext(out var psion, out var psionicComponent, out _))
         {
             if (!_mobStateSystem.IsAlive(psion) || HasComp<PsionicInsulationComponent>(psion))
                 continue;
 
             _stunSystem.TryParalyze(psion, TimeSpan.FromSeconds(5), false);
```

Comment:
Similarly to https://github.com/Simple-Station/Einstein-Engines/pull/731/files#r1718973998, though you didn't touch this line, you might want to make it a const.

Reply:
It'll probably be better to define that 5 in the NoosphericZapRuleComponent(Gamerules have components!). We generally don't want to put Consts in System files, since doing so goes against a bit of the design philosophy around ECS. It's fine if the gamerule has the 5 listed as a const, because then someone can go into the associated yml file and change it there without needing to recompile the game. 

Another thing to note is that you are actually onto something about picking through my code and pointing out anywhere I've left a stray number. The only acceptable numbers that should ever be used in system files directly are 0 and 1, when in the context of building logic around AdditiveIdentity and MultiplicativeIdentity.